CONTRIBUTING.md

```diff
@@ -53,18 +53,18 @@ the mailing list have a tendency to fall through the cracks.
 
 ### ChangeLog
 
-All PRs need to have a one-line description in the initial pull request body.
+All PRs need to have a one-line summary in the initial pull description.
```

Comment:
I'd really stress here that it's the PR message, which can be different to the commit message:

```
The PR description can be different to the last commit's message
```

Reply:
I gave explicit instructions on how to edit the PR description. If possibe, I'd like to avoid negative documentation, i.e. document where things do *not* go. Do you think an example screenshot of a ChangeLog line would be helpful to contributors?